README.md

```diff
@@ -340,6 +327,61 @@ Note how this example responds to various types of navigations:
 
 Notice also how by passing through the `AbortSignal` found in `e.signal`, we ensure that any aborted navigations abort the associated fetch as well.
 
+#### Example: async transitions with special back/forward handling
+
+Sometimes it's desirable to handle back/forward navigations specially, e.g. reusing cached views by transitioning them onto the screen. This can be done by branching as follows:
+
+```js
+appHistory.addEventListener("navigate", e => {
```

Comment:
I think you need async here?

```suggestion
appHistory.addEventListener("navigate", async (e) => {
```

Reply:
Oh, the problem here is I need to wrap this all in respondWith.